tetris.py

```diff
@@ -154,65 +158,19 @@ def draw_block(self):
                     fill=self.color_dict[letter],
                 )
 
-        for x, y in self.tetris_game.get_block_shape(
-            self.tetris_game.upcoming_block_shape,
-            self.tetris_game.upcoming_block_center,
-            0,
-        ):
-            self.canvas_small.create_rectangle(
-                x * square_size,
-                y * square_size,
-                x * square_size + square_size,
-                y * square_size + square_size,
-                tags="block",
-                fill=self.color_dict[self.tetris_game.upcoming_block_shape],
-            )
-
-    def move_block(self):
-        """
-        Has the responsibility to call current_block_mover() and draw_block() to
-        simulate the blocks moving downwards on the canvas
-        """
-        self.tetris_game.current_block_mover()
-        self.draw_block()
-        self.canvas.after(game_speed, self.move_block)
-
-    def move_block_left(self, event):
-        self.tetris_game.user_input_left()
-        self.draw_block()
-
-    def move_block_right(self, event):
-        self.tetris_game.user_input_right()
-        self.draw_block()
-
-    def rotate_block(self, event):
-        self.tetris_game.block_rotator()
+    def letter_generator(self):
+        self.block_letter = random.choice(block_letter)
         self.draw_block()
 
-    def timer(self):
-        game_time = time.time() - self.start_time
-        return f"{int(game_time / 60):02d}:{int(game_time % 60):02d}"
-
-
-class TetrisGame:
-    def __init__(self):
-        self.landed_blocks = {}  # e.g. {'L': [(1, 2), (3, 4)]}
-        self.upcoming_block_shape = None
-        self.new_block()
-
     def new_block(self):
-        """
-        Chooses a random block from "blocks" and assigns it to
-        self.current_block
-        """
-        if self.upcoming_block_shape is None:
-            self.current_block_shape = random.choice(shape_names)
-        else:
-            self.current_block_shape = self.upcoming_block_shape
         self.current_block_center = (int(game_width / 2), -2)
-        self.upcoming_block_shape = random.choice(shape_names)
-        self.upcoming_block_center = (2, 1)
+        self.block_letter = self.small_board.block_letter
+        self.small_board.block_letter = random.choice(block_letter)
+        self.draw_block()
+        print(self.small_board.block_letter)
         self.rotate_counter = 0
+        self.get_block_shape(self.block_letter, self.current_block_center, self.rotate_counter)
+
 
     def get_block_shape(self, block_shape, block_center, rotate_counter):
```

Comment:
This method should use `self.block_shape` etc:

```suggestion
    def get_block_shape(self):
```